### PR TITLE
[8.1] removed setup parameter from WMS

### PR DESCRIPTION
**Pull request by fstagni**
This will require DB changes.
And quite some tests for backward compatibility.

- [x] add to wiki - in https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-8.1#wms

BEGINRELEASENOTES

*WMS
CHANGE: removed Setup

ENDRELEASENOTES

### CODE REVIEW

**Comment by chrisburr** [comment]
Converting to draft until we're ready to make a release branch for 8.0.

**Comment by fstagni** [comment]
Review please.

File: src/DIRAC/WorkloadManagementSystem/DB/SandboxMetadataDB.py [hunk 1]
```diff
@@ -261,43 +257,39 @@ def __filterEntitiesByRequester(self, entitiesList, entitiesSetup, requesterName
             return result
         return S_OK([row[0] for row in result["Value"]])
 
-    def unassignEntities(self, entitiesDict, requesterName, requesterGroup):
+    def unassignEntities(self, entities, requesterName, requesterGroup):
         """
         Unassign jobs to sandboxes
-        entitiesDict = { 'setup' : [ 'entityId', 'entityId' ] }
+        entities = ['entityId_1', 'entityId_2']
```
Comment (atsareg) [inline src/DIRAC/WorkloadManagementSystem/DB/SandboxMetadataDB.py]:
```suggestion
        :param list entities: list of entities to unassign
```

File: src/DIRAC/WorkloadManagementSystem/DB/TaskQueueDB.py [hunk 2]
```diff
@@ -834,13 +824,11 @@ def __generateTQMatchSQL(self, tqMatchDict, numQueuesToGet=1, negativeCond=None)
...
-            fullTableN = "`tq_TQTo%ss`" % field
+            fullTableN = f"`tq_TQTo{field}s`"
 
             sqlCondList.append(
                 self.__generateSQLSubCond(
-                    "%%s not in ( SELECT %s.Value \
-                                                      FROM %s \
-                                                      WHERE %s.TQId = tq.TQId )"
+                    "%%s not in ( SELECT %s.Value FROM %s WHERE %s.TQId = tq.TQId )"
                     % (fullTableN, fullTableN, fullTableN),
```
Comment (atsareg) [inline src/DIRAC/WorkloadManagementSystem/DB/TaskQueueDB.py]:
```suggestion
                    f"%%s not in ( SELECT {fullTableN}.Value FROM {fullTableN} WHERE {fullTableN}.TQId = tq.TQId )",
```

**Review by atsareg — DISMISSED** [review]

**Pull request merged**